.github/workflows/project-codeflare-release.yml

```diff
@@ -72,7 +72,7 @@ jobs:
 
     - name: Release MCAD
       run: |
-        gh workflow run mcad-release.yml --repo ${{ github.event.inputs.codeflare-repository-organization }}/multi-cluster-app-dispatcher --ref go-1.20 --field tag=${{ github.event.inputs.mcad-version }}
+        gh workflow run mcad-release.yml --repo ${{ github.event.inputs.codeflare-repository-organization }}/multi-cluster-app-dispatcher --ref go-1.21 --field tag=${{ github.event.inputs.mcad-version }}
```

Comment:
This can remain go-1.20. That is the branch name, there currently isn't a go-1.21 branch in `multi-cluster-app-dispatcher`

Reply:
reverted this change.